pkg/asset/agent/image/releaseimage.go

```diff
@@ -33,6 +33,7 @@ func releaseImageFromPullSpec(pullSpec, arch string) (releaseImage, error) {
 		return releaseImage{
 			ReleaseVersion: versionString,
 			Arch:           arch,
+			Archs:          archs,
```

Comment:
@jeffdyoung this point it's not clear to me. What's the requirement to have separate fields `Archs` and `Arch`? Would make sense to have just one field `Archs` to cover all the cases?

Reply:
Looks like we'll need both for now:
```
level=fatal msg="failed to create Versions handlers" func=main.main.func1 file="/src/cmd/main.go:237" error="Missing value in ReleaseImage for 'cpu_architecture' field"
```

Reply:
This isn't an exhaustive list, but I see both used in various parts of the assisted-service:
https://github.com/openshift/assisted-service/blob/master/models/infra_env.go#L39
https://github.com/openshift/assisted-service/blob/master/models/infra_env_create_params.go#L40
https://github.com/openshift/assisted-service/blob/master/models/os_image.go#L26
https://github.com/openshift/assisted-service/blob/master/models/release_image.go#L27-L30
https://github.com/openshift/assisted-service/blob/master/models/openshift_version.go#L25